src/legacy/core_plugins/metrics/public/components/lib/tick_formatter.js

```diff
@@ -18,36 +18,61 @@
  */
 
 import handlebars from 'handlebars/dist/handlebars';
-import { durationInputOptions } from './durations';
-import { capitalize, isNumber } from 'lodash';
+import { isNumber } from 'lodash';
 import { fieldFormats } from 'ui/registry/field_formats';
 
-const durationsLookup = durationInputOptions.reduce((acc, row) => {
-  acc[row.value] = row.label;
-  return acc;
-}, {});
+const inputFormat = {
+  'ps': 'picoseconds',
+  'ns': 'nanoseconds',
+  'us': 'microseconds',
+  'ms': 'milliseconds',
+  's': 'seconds',
+  'm': 'minutes',
+  'h': 'hours',
+  'd': 'days',
+  'w': 'weeks',
+  'M': 'months',
+  'Y': 'years',
+};
+
+const outputFormat = {
+  'humanize': 'humanize',
+  'ms': 'asMilliseconds',
+  's': 'asSeconds',
+  'm': 'asMinutes',
+  'h': 'asHours',
+  'd': 'asDays',
+  'w': 'asWeeks',
+  'M': 'asMonths',
+  'Y': 'asYears',
+};
```

Comment:
I think we should avoid duplicate the input and output parameters, maybe it's better to specify the `moment` function to call into the `durationBaseOptions` et similar array.

Reply:
@markov00  I don't understand about which 'diplicate' you are talking? It's a different sets of data also outputFormats starts from 'as' but it's not true for all cases.